### PR TITLE
Draft: Add option for dark mode

### PR DESCRIPTION
**Pull request by ropapermaker**
Currently working on adding an option to enable dark mode as mentioned in #17819.

Currently I have only created a new styling sheet for dark mode.

Dark mode UI snippets:
<details>
  <summary>Old dark colors snippets</summary>
  
  
  
</details>
<details>
  <summary>New dark colors snippets</summary>
  
  
  
  
</details>

Tested on Firefox and Chromium
TODO:
- [x] update drop-downs, input text to dark colors
- [ ] refactor classes and variables to achieve uniformity
- [ ] add option in settings or navbar to change themes

### CODE REVIEW

File: ui/app/styles/core.scss [hunk 1]
```diff
@@ -126,3 +126,6 @@
 
 // bulma-free-zone
 @import './components/icon';
+
+// dark mode
+@import './core/dark-mode';
```
Comment (darpham) [inline ui/app/styles/core.scss:131]: EOF newline?

Comment (darpham) [reply ui/app/styles/core.scss:131]: Same nit `ui/app/styles/core/dark-mode.scss`

Comment (ropapermaker) [reply ui/app/styles/core.scss:131]: was not aware of this, fixing on next update. thanks

**Comment by ropapermaker** [comment]
> Hi 👋 @ropapermaker. Thank you for working on this. I wanted to give you a heads up before you put more effort into this PR that in the 1.14 release there will be significant .scss file changes. We're removing Bulma—a third party css framework—and doing some internal restructuring or our style files. The file changes will likely not be compatible with your work here. Dark-mode is on our radar as it's a highly requested feature from the community.

Hi @Monkeychip. Thank you for letting me know. I guess this PR can be closed since it would not contribute to the future work on theming. If there is any help needed regarding this I would be more than happy to contribute :D.

**Comment by VioletHynes** [comment]
Closing as per submitter request - the changes we're making to support theming will render a lot of this incompatible unfortunately. Thanks for working on this!

**Comment by anvme** [comment]
for all researchers
420616970-ecbb1c52-9de4-4a4b-97be-60e53824b9fe

**Pull request closed without merging**